_posts/2016-04-01-fintech-demo-day.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: "FinTech Sandbox Demo Day"
+description: "A short recap of the FinTech Sandbox's Demo Day"
+category:
+tags: [FinTech, conference]
+author: vikram_ramakrishnan
+---
+{% include JB/setup %}
+
+Last week, [Greenfield](www.greenfieldhq.com) attended the [FinTech Sandbox](http://FinTechsandbox.org/)'s Demo Day, which highlighted
```

Comment:
I think this might need to be the full url like `http://www.greenfieldhq.com`. When I click on the link from the preview on GitHub, it just treats this like a relative path and goes to like `http://www.github.com/www.greenfieldhq.com`